examples/compression/main.go

```diff
@@ -8,8 +8,10 @@ import (
 
 	"github.com/momentohq/client-sdk-go/auth"
 	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/config/compression"
 	"github.com/momentohq/client-sdk-go/config/logger/momento_default_logger"
 	"github.com/momentohq/client-sdk-go/config/middleware"
+	"github.com/momentohq/client-sdk-go/config/middleware/impl"
```

Comment:
```suggestion
	middlewareImpl "github.com/momentohq/client-sdk-go/config/middleware/impl"
```

I'd recommend using an import alias here for readability.